.github/workflows/ci-airflow.yml

```diff
@@ -39,6 +39,9 @@ jobs:
       - name: Install dependencies
         run: pip install -r build/requirements-airflow-dev.txt
 
+      - name: Initialize airflow database for loading variables
```

Comment:
Adding this extra step will make the tests being non self-contained. There are a couple better approaches, please see `test_monitoring.py`.